.github/workflows/build.yml

```diff
@@ -7,13 +7,13 @@ jobs:
     strategy:
       matrix:
         platform: [ ubuntu-latest, macos-latest, windows-latest ]
-        toolchain: [ stable, 1.48.0 ]
+        toolchain: [ stable, 1.49.0 ]
         include:
           - toolchain: stable
             check-fmt: true
         exclude:
           - platform: macos-latest
-            toolchain: 1.48.0
+            toolchain: 1.49.0
```

Comment:
> We usually (informally) require reviews for ldk-sample

Ugh, sorry, that was kind of an accident. Happy to revert anything in a follow-up.

> Why did you bump the toolchain? 1.48 should remain our MSRV.

Because tokio recently bumped its MSRV from 1.48.0 to 1.49.0. As it is probably the major dependency of this project, I thought it would make sense to bump MSRV in its footsteps to ensure compatibility, rather than pinning a particular minor version. Happy to revert that in a follow-up though, we can probably also pin to 1.20.x LTS.